core/src/main/scala/kafka/admin/ConsumerGroupCommand.scala

```diff
@@ -54,8 +54,12 @@ object ConsumerGroupCommand {
     opts.checkArgs()
 
     val consumerGroupService = {
-      if (opts.options.has(opts.newConsumerOpt)) new KafkaConsumerGroupService(opts)
-      else new ZkConsumerGroupService(opts)
+      if (opts.useOldConsumer) {
+        Console.err.println("Old consumer is deprecated and will be removed in a future release. Consider using the new consumer.")
```

Comment:
Changed to `Using the ConsumerGroupCommand with old consumer is deprecated and will be removed in a future release. Consider using the new consumer.`. Is this ok?